Tests?

Hey, I am not sure I understand how the tests are supposed to work. Also, once I figure this out, would you be open for having the tests run on e.g. Travis-CI?

Anyway, I `npm install`'d in the root of the repo, then `node test/internal/index.js`:

```
module.js:340
    throw err;
          ^
Error: Cannot find module 'socketcluster-client'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/path/socketcluster/test/internal/index.js:3:16)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
```

--- a/package.json
+++ b/package.json
@@ -39,5 +39,8 @@
     "scalable"
   ],
   "license": "MIT",
-  "readmeFilename": "README.md"
+  "readmeFilename": "README.md",
+  "devDependencies": {
+    "socketcluster-client": "5.x.x"
+  }
 }

